model/schema/expression.yaml

```diff
@@ -23,16 +23,24 @@ classes:
       Specimens are defined by their AGM and/or alleles (or gel lane at MGI).
     slots:
       - curie
-      - references
+      - single_reference
       - biological_entity_assayed
       - assay_used
       - reagents_used
       - specimen_genomic_model
       - specimen_alleles
       - condition_relations
-    slot_usage:
-      references:
-        multivalued: false    # Should I call this slot just "reference" then?
+    rules:
+      - postconditions:
+          any_of:
+            - slot_conditions:
+                biological_entity_assayed:
+                  required: true
+            - slot_conditions:
+                reagents_used:
+                  required: true
+          description: At least one of biological_entity_assayed or reagents_used should be populated.
+
```

Comment:
Further down in the file, `reagents_used` is not listed as `required` but `biological_entity_assayed` is listed as `required`. Does that need to change?

Reply:
yep! Good catch.